mobile/src/main/java/ch/epfl/sdp/mobile/ui/Graphics.kt

```diff
@@ -64,6 +64,9 @@ val PawniesIcons.LocalPlay
 val PawniesIcons.OnlinePlay
   @Composable get() = painterResource(id = R.drawable.online_play)
 
+val PawniesIcons.Close
+  get() = Icons.Default.Close
+
```

Comment:
We already have a close icon, can we reuse that ? 

https://github.com/epfl-SDP/android/blob/d4ba1aad4a83a509f5d9c020e7fea45cee7d08ca/mobile/src/main/java/ch/epfl/sdp/mobile/ui/Graphics.kt#L20-L22

Reply:
Yes indeed, should I reformat the name then so that is more general like simply `PawniesIcons.Close`

Reply:
Yes please